data/creaturescripts/monster/minion_gaz_haragoth_vortex.lua

```diff
@@ -0,0 +1,26 @@
+local function removeTeleport(position)
+    local teleportItem = Tile(position):getItemById(20121)
+    if teleportItem then
+        teleportItem:remove()
+    end
+end
+
+local minionDeath = CreatureEvent("MinionGazDeath")
+
+function minionDeath.onDeath(creature, corpse, killer, mostDamageKiller, unjustified, mostDamageUnjustified)
+    local targetMonster = creature:getMonster()
+    if not targetMonster then
+        return true
+    end
+
+    local deathPosition = creature:getPosition()
+    local item = Game.createItem(20121, 1, deathPosition)
+
+    item:setActionId(33542)
+
+    addEvent(removeTeleport, 1 * 60 * 1000, deathPosition)
```

Comment:
You can change for use position lib function: https://github.com/opentibiabr/otservbr-global/blob/6d43e5b20200ba32010f75ca87a8898c41aedca3/data/lib/core/position.lua#L342

```suggestion
    addEvent(deathPosition:removeTeleport, 1 * 60 * 1000, 20121)
```

Please, test the change.

Reply:
I just tried this but it doesn't work